src/main/java/adris/altoclef/tasks/misc/speedrun/CollectBlazeRodsTask.java

```diff
@@ -17,25 +21,45 @@
 import net.minecraft.util.math.Vec3d;
 import net.minecraft.world.RaycastContext;
 
-public class CollectBlazeRodsTask extends ResourceTask {
 
+public class CollectBlazeRodsTask extends ResourceTask {
     private static final double SPAWNER_BLAZE_RADIUS = 32;
-
     private static final int TOO_MANY_BLAZES = 5;
-    private static final double TOO_LITTLE_HEALTH_BLAZE = 5;
-
-    private BlockPos _foundBlazeSpawner = null;
-
-    private final int _count;
+    private static final double TOO_LITTLE_HEALTH_BLAZE = 5; // TODO: 2021-05-22 instead of caring about health, maybe just make
```

Comment:
The bot already dodges projectiles, this is just a backup "oh no we gotta run away" value or something like that. (shield is a good idea but would be handled separately)

I also have never seen the bot run away from blazes or ever consider its health. Even if it did, right now it most likely would be at its detriment (it merely wanders, doesn't flee) so that's a fix for later.